.github/workflows/scripts/countline.py

```
@@ -315,24 +316,18 @@ def main():
     if bool(errors_found) is True:
-        print(
-            """
+        print("""
 The {} files listed above have more than {} lines of code.
 
 Please fix this. It is a pre-requisite for pull request approval.
-""".format(
-                file_count, args.lines
-            )
-        )
+""".format(file_count, args.lines))
```

Ruff (0.14.13)

Use f-string instead of `format` call

Convert to f-string

(UP032)